src/style.css

```diff
@@ -1,3 +1,81 @@
+:root {
+  --default-color-link: #000;
+  --color-active-link: #00acdc;
+  --background-header: #fff;
+  --box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
It's recommended to add a fallback font-family. This is used in case the main font (Roboto here) fails to load or is not available on the user's system. For example, you can use a generic font family like 'sans-serif' as a fallback.